ansible/roles/dashmate/templates/dashmate.json.j2

```diff
@@ -1,5 +1,5 @@
 {
-  "configFormatVersion": "{{ dashmate_version | default('999.0.0') }}",
+  "configFormatVersion": "{{ dashmate_version | default(dashmate_config_version | default('999.0.0')) }}",
```

Comment:
Why do you we want to configure this?

Reply:
Because when we install from branch, we don't have dashmate_version available and it will back off to `999.0.0` which will break template idempotency. On the first dashmate run, config version will be overwritten with the package default version.

Reply:
I see. Yes you right. Would it be better to get dashmate version form dashmate itself (`dashmate --version` or `package.json`) so you won't need to configure it manually. `999.0.0` workaround won't be necessary as well.